NET-Core-ASP-API.Tests/Services/PeopleServiceTests.cs

```diff
@@ -60,5 +61,60 @@ public void ReturnsNull()
                 result.ShouldBeNull();
             }
         }
+
+        [TestFixture]
+        public class CreateShould : PeopleServiceTests
+        {
+            private readonly static string _firstName = "Eve";
+            private readonly static string _surname = "Harper";
+            private readonly static int _age = 22;
+            private readonly Person _personCreationData = new Person { FirstName = _firstName, Surname = _surname, Age = _age };
+            private readonly Person _createdPerson = new Person { ID = 1, FirstName = _firstName, Surname = _surname, Age = _age };
+
+            [Test]
+            public void CallCreatePerson()
+            {
+                var result = service.CreatePerson(_personCreationData);
+
+                peopleRepoMock.Verify(q => q.CreatePerson(_personCreationData), Times.Once());
+            }
+
+            [Test]
+            public void ReturnCreatedPerson()
+            {
+                peopleRepoMock.Setup(q => q.CreatePerson(_personCreationData)).Returns(_createdPerson);
+
+                var result = service.CreatePerson(_personCreationData);
+
+                result.ShouldBeOfType<Person>();
+                result.ShouldBe(_createdPerson);
+            }
+
+            [Test]
+            public void ReturnConflict()
```

Comment:
Could rename 'ThrowDuplicateRecordException'?

Reply:
Oops. Copy and paste fat finger. Corrected